### ISSUE TITLE
Wrong link associated with CF deployer

### ISSUE THREAD

**Issue by sabbyanandan** [spring-cloud/spring-cloud-dataflow#339]
As a user, I'm trying to follow the instructions from [reference guide](http://docs.spring.io/spring-cloud-dataflow/docs/1.0.0.M1/reference/html/getting-started-deploying-spring-cloud-dataflow.html#_deploying_on_cloud_foundry) and I ran into following error.

```
2016-01-28T13:28:24.98-0800 [APP/0]      OUT Caused by: java.io.IOException: Cannot run program "java" (in directory "/home/vcap/tmp/spring-cloud-data-flow-1734416835671165682/ticktock-1454016504972/ticktock.log"): error=2, No such file or directory
2016-01-28T13:28:24.98-0800 [APP/0]      OUT at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048) ~[na:1.8.0_65-]
2016-01-28T13:28:24.98-0800 [APP/0]      OUT at org.springframework.cloud.dataflow.admin.spi.local.OutOfProcessModuleDeployer$Instance.<init>(OutOfProcessModuleDeployer.java:237) ~[app/:na]
2016-01-28T13:28:24.98-0800 [APP/0]      OUT at org.springframework.cloud.dataflow.admin.spi.local.OutOfProcessModuleDeployer$Instance.<init>(OutOfProcessModuleDeployer.java:211) ~[app/:na]
```

The link is pointing to `local` as opposed to `cf` deployer artifact.

**Comment by trisberg** [comment]
Those are the instructions for SCDF 1.0.0.M1 back in September 2015. Is that the version you are looking to use?

**Comment by sabbyanandan** [comment]
Ah, haa. Unfortunately, someone specifically tried the M1 release. Looks like the M2 [release docs](http://docs.spring.io/spring-cloud-dataflow/docs/1.0.0.M2/reference/html/getting-started-deploying-spring-cloud-dataflow.html#_deploying_admin_app_on_cloud_foundry) are pointing to the right repo link. I'll close this issue.

**Comment by trisberg** [comment]
Actually, the M1 CF docs have the wrong file name -- they have M2 as the version:

http://docs.spring.io/spring-cloud-dataflow-admin-cloudfoundry/docs/1.0.0.M1/reference/html/getting-started.html

**Comment by trisberg** [comment]
The CF admin should be:

    wget http://repo.spring.io/snapshot/org/springframework/cloud/spring-cloud-dataflow-admin/1.0.0.BUILD-SNAPSHOT/spring-cloud-dataflow-admin-cloudfoundry-1.0.0.BUILD-SNAPSHOT.jar

**Comment by trisberg** [comment]
Same issue with the version for spring-cloud-dataflow-admin-yarn. We changed the version to M1 just before the release. We should replace the on-line docs with corrected ones. Also, we don't need CF and YARN in the main doc since they have their own docs.

**Comment by sabbyanandan** [comment]
> we don't need CF and YARN in the main doc since they have their own docs

+1

@markpollack - thoughts?

**Comment by markpollack** [comment]
I already removed them after the release....

See http://docs.spring.io/spring-cloud-dataflow/docs/1.0.x-SNAPSHOT/reference/html/

**Comment by markpollack** [comment]
One can log in to change the ref docs for M1 and make a manual deletion in the html.

**Comment by markpollack** [comment]
Strange as I thought I had Trevor correct the url for downloading CF admin... Guess something got confused.  Need to put in variables substitution in some spots so that we can reduce errors in the future.

**Comment by markpollack** [comment]
Ok, fixed the listing of other impls in the dataflow M2 docs and fixed the wrong version number in the M1 cloud foundry admin for both html and htmlsingle

I didn't find the wrong version in the M1 for YARN.  It lists  

```wget http://repo.spring.io/milestone/org/springframework/cloud/dist/spring-cloud-dataflow-admin-yarn-dist/1.0.0.M1/spring-cloud-dataflow-admin-yarn-dist-1.0.0.M1.zip```

on http://docs.spring.io/spring-cloud-dataflow-admin-yarn/docs/1.0.0.M1/reference/html/_deploying_on_yarn.html

@trisberg What did I miss?

**Comment by markpollack** [comment]
Fixed `cd spring-cloud-dataflow-admin-yarn-1.0.0.M2` to ref M1 on http://docs.spring.io/spring-cloud-dataflow-admin-yarn/docs/1.0.0.M1/reference/htmlsingle/#_deploying_on_yarn
